llvm/lib/Transforms/IPO/GlobalOpt.cpp

```diff
@@ -2001,6 +2012,8 @@ static bool processInternalGlobal(
       if (isa<UndefValue>(GV->getInitializer())) {
         // Change the initial value here.
         GV->setInitializer(SOVConstant);
+        // Change the GV digest value.
```

Comment:
This isn't repo-specific code so the comment should probably mention that the call is for the benefit of the repo (same applies to the other occurrences).

Reply:
Fixed, thanks.